### PR TITLE
doc: proofread changelogs 2.11-rc part 1

### PR DESCRIPTION
**Pull request by p7nov**
Fix grammar, wording, and formatting

Resolves https://github.com/tarantool/doc/issues/3287

NO_CHANGELOG=changelog
NO_DOC=changelog
NO_TEST=changelog

### CODE REVIEW

File: changelogs/unreleased/gh-6833-encode-body-http-request.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## feature/lua/http client
 
-* HTTP client is able to encode Lua objects automatically when it is possible.
-  Encoding depends on content type passed to HTTP request and following types
-  are supported: "application/json", "application/yaml" and
-  "application/msgpack". Now user can define his own encoding function (gh-6833).
+* Now the HTTP client is able to encode Lua objects automatically when it is possible.
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6833-encode-body-http-request.md:3]: Nit: please, fit the lines to 80 symbols.

Comment (p7nov) [reply changelogs/unreleased/gh-6833-encode-body-http-request.md]: Done. Thanks

Comment (Totktonada) [reply changelogs/unreleased/gh-6833-encode-body-http-request.md]: That's done, but now the entry is not a markdown list items -- it lacks the asterisk.

Comment (p7nov) [reply changelogs/unreleased/gh-6833-encode-body-http-request.md]: Ouch, my bad. Fixed.

**Review by kyukhin — APPROVED** [review]

**Comment by locker** [comment]
image

@p7nov In contrast to tarantool/doc, we use rebase-merging in tarantool/tarantool (no automatic squashing). In future please squash commits and force-push (`git commit --amend`) instead of piling fixes on top of each other.

@kyukhin note.

**Pull request merged**